src/workloads/scale/LLTMixed.yml

```diff
@@ -148,11 +148,11 @@ GlobalDefaults:
           Level: majority
 
   # Low Baseline / Benchmark
-  LowGlobalRate: &LowGlobalRate 400 per 1 second
+  LowGlobalRate: &LowGlobalRate 100 per 1 second
```

Comment:
No, I tried match the load in the previous version of LLTMixed. As we have 4 actors now instead of one, I used a quarter of the original rate. We are pretty close to the max - user_time got to 7.5 cpus at its peak (and there was a peak there) and around 5 cpu at the beginning/end of high load (of 8 cpus we do use). I will run more - but I guess it would be more stress testing and probably should be separate from this PR.